src/utils/get-command-spec.test.ts

```diff
@@ -23,6 +23,12 @@ describe('#getCommandSpec()', () => {
 		);
 	});
 
+	it('Complains about invalid spec JS', async () => {
+		await expect(getCommandSpec(path.join(testFileTrees, 'bad-structure', 'cli', 'invalid-data-js'))).rejects.toThrow(
+			'Encountered this error while importing the spec file at'
+		);
+	});
```

Comment:
This test has the same descriptor as the previous one. It should have a unique descriptor.